### PR TITLE
fix comment typo, change paths ini file name and added quick README.md

### PR DESCRIPTION
**Pull request by Cyberface**
I think there was a small typo in the comments for this line of code.
Agree?

### CODE REVIEW

File: nrutils/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# nrutils_dev readme
+
+## notes on installation
+
+```
+python setup.py install
+```
+
+add the path to ```nrutils_dev``` to ```$PYTHONPATH```
```
Comment (Galadirith) [inline nrutils/README.md]: @Cyberface This shouldn't be necessary with #3. Sweet markdown :smile:

Comment (Cyberface) [inline nrutils/README.md]: @Galadirith which parts won't be necessary sorry? just line 9 right?

Comment (Galadirith) [inline nrutils/README.md]: @Cyberface Sorry Seb I didn't explain properly. Yeh just line 9. I believe the problem was that `MANIFEST.in` specifies files to be included in a distribution (not sure how to do that 😭 ) but isn't used when you run `setup.py install`.

Comment (Cyberface) [inline nrutils/README.md]: Ah! I see how these comments work now :). I thought you were talking about the lines 1-9 :)
Ok removed that line. Thanks!

**Pull request merged**